static/css/section/_cloud.scss

```diff
@@ -178,15 +178,15 @@
     .row-maas {
 
       @media only screen and (min-width : $breakpoint-medium) {
-        background-image: url('#{$asset-server}1066e824-cloud_maas.jpg?op=region&rect=0,0,300,500');
-        background-position: right top;
-        background-repeat: no-repeat;
+        //background-image: url('#{$asset-server}1066e824-cloud_maas.jpg?op=region&rect=0,0,300,500');
```

Comment:
Could you remove these commented-out lines? Or add a note saying why you're leaving them in?